tools/StaticAnalysis/Exceptions/Az.Network/SignatureIssues.csv

```diff
@@ -303,4 +303,12 @@
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NewAzureFirewallPolicyFilterRuleCollectionCommand","New-AzFirewallPolicyFilterRuleCollection","1","8100","New-AzFirewallPolicyFilterRuleCollection Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NewAzureFirewallPolicyNatRuleCollectionsCommand","New-AzFirewallPolicyNatRuleCollection","1","8100","New-AzFirewallPolicyNatRuleCollection Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NewAzureFirewallPolicyNetworkRuleCommand","New-AzFirewallPolicyNetworkRule","1","8100","New-AzFirewallPolicyNetworkRule Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.RemoveAzureFirewallPolicyCommand","Remove-AzFirewallPolicy","1","8510","Cmdlet 'Remove-AzFirewallPolicy' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NewAzureFirewallPolicyRuleCollectionGroupCommand","New-AzFirewallPolicyRuleCollectionGroup","1","8510","Cmdlet 'New-AzFirewallPolicyRuleCollectionGroup' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.RemoveAzureFirewallPolicyRuleGroupCommand","Remove-AzFirewallPolicyRuleCollectionGroup","1","8510","Cmdlet 'Remove-AzFirewallPolicyRuleCollectionGroup' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.SetAzureFirewallPolicyRuleGroupCommand","Set-AzFirewallPolicyRuleCollectionGroup","1","8510","Cmdlet 'Set-AzFirewallPolicyRuleCollectionGroup' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
```

Comment:
These 4 lines are not false positive. Please address the issues and remove them from suppression.

Reply:
Removed them and addressed these